bot/seasons/valentines/movie_generator.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import random
+from os import environ
+from urllib import parse
+
+import aiohttp
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class MovieGenerator:
+    """
+    a cog that gives random romance movie suggestion to a user
+
+    """
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def romancemovie(self, ctx):
+        """
+
+        randomly selects romance movie and displays info about it
```

Comment:
This line should be capitalised and be grammatically correct
```suggestion
        Randomly selects a romance movie and displays information about it
```